docs/source/compute-git.md

```diff
@@ -1,13 +1,53 @@
+For readability, we will add a couple links for each topic we wish to guide you through.
+
+## Table of Contents
+1. Starting with git/GitHub
+2. Repositories
+    - Forks
+    - Clones
+3. Branches
+4. Updating your repository
+    - Commits
+    - Merge
+    - Pull requests
+5. Advanced topics
+
+## 1. Starting with git/GitHub
+* This [tutorial](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners) explains the differences between git and GitHub and walks its readers through all the basic steps to get started with GitHub.
+* Here is another [tutorial](https://www.freecodecamp.org/news/git-and-github-for-beginners/) that introduced git and GitHub for beginners.
+
+## 2. Repositories:
+* The first place we recommend is the GitHub group's [documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories) of repository management.
+* Here is a great [guide](https://github.blog/2023-08-04-a-checklist-and-guide-to-get-your-repository-collaboration-ready/#get-started) for understanding the resposibilities for maintaining a repository.
+
+There are many things you can do with a GitHub repository. The 2 major actions would be to fork the repository or clone it.
+We recommend following this [link](https://www.theserverside.com/answer/Git-fork-vs-clone-Whats-the-difference) to understand the difference between these two actions.
+### Forks:
+* This workflow [tutorial](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow#:~:text=The%20main%20advantage%20of%20the,push%20to%20the%20official%20repository.) provides a great explanation for git forks and where they stand in the entire workflow when contributing to a project.
+
+### Clones:
+* This workflow [tutorial](https://www.warp.dev/terminus/git-clone-ssh) will be useful when beginning to clone git repositories to understand the differences between using HTTPS and SSH and how we set up SSH for cloning.
+
+## 3. Branches
+* This is a great [breakdown](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell) of how branches can be created and how they differ from the main/master of git repositories.
+* We also recommend using the GitHub [documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches#) to understand more on branches.
+
+## 4. Updating your repository
+After gaining a basic understanding of the topics covered above, you are prepared to start making changes to your GitHub repository.
+The process typically consists of committing changes and creating pull requests to merge changed branches.
+
+### Commits:
+* Here is a great [guide](https://github.blog/2022-06-30-write-better-commits-build-better-projects/) for understanding what makes a good git commit. It goes over what the messages should look like and when to make each commit.
+
+### Pull requests:
```

Comment:
there was a bullet "merge" in the TOC, did you decide to drop it?

Reply:
Yeah I decided that the pull requests might include the information I wanted to cover so I didn't know what to add to make "merge" have enough